tests/example_test.go

```diff
@@ -121,6 +121,11 @@ func TestExample(t *testing.T) {
 		commit  string
 		skip    stringSet
 	}{
+		{
+			example: "https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner",
+			// TODO[pulumi/pulumi-converter-terraform#212]: Hetzner cloud has various conversion issues.
+			skip: allLanguages,
+		},
```

Comment:
Why are we adding it if it's always skipped? Would it be better to add it when we fix it?

Reply:
Right now we're actively trying to unblock these, I just want to make sure these cases aren't forgotten and tracked in the code as well.  I can remove it and just use the bug but don't want to accidentally leave this user behind

Reply:
Yeh most of the others below are all skipped as well, it's fine to add to track it